src/applications/vaos/components/LoadingOverlay.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+export default function LoadingOverlay({ show, message }) {
+  useEffect(
+    () => {
+      if (show) {
+        document.body.classList.add('modal-open');
+      } else {
+        document.body.classList.remove('modal-open');
```

Comment:
Copied over from the Modal component.  Adds `.modal-open` class to `<body>` which prevents scrolling while overlay is showing.